### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by kianmeng**
Found via `codespell -S *.svg`

### CODE REVIEW

File: financeager/exceptions.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-"""Module defining expections of the financeager application."""
+"""Module defining expectations of the financeager application."""
```
Comment (pylipp) [inline financeager/exceptions.py:1]: Oh this should mean `exceptions` :sweat_smile:

File: test/test_pocket.py [hunk 2]
```diff
@@ -519,7 +519,7 @@ def test_invalid_recurrent_entry(self):
                 {
                     "name": "rent",
                     "value": -400,
-                    "frequency": "yaerly",
```
Comment (pylipp) [inline test/test_pocket.py:522]: This one's actually on purpose :)

**Review by pylipp — CHANGES_REQUESTED** [review]
thanks @kianmeng for the fixes!

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**